go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/eagraf/habitat-new
 
-go 1.21
```

Comment:
I think we gotta run `go mod tidy` to recalculate the lock file

Reply:
Running doesn't seem to change anything